community/portworx/templates/portworx-ds.yaml

```diff
@@ -22,23 +22,16 @@
 {{- $etcdCA := .Values.etcd.ca | default "none" }}
 {{- $etcdCert := .Values.etcd.cert | default "none" }}
 {{- $etcdKey := .Values.etcd.key | default "none" }}
-{{- $consulToken := .Values.consul.token | default "none" }}
 {{- $etcdEndPoints := .Values.kvdb }}
-{{- $misc := .Values.misc | default "" | split " " }}
 
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: portworx
   namespace: kube-system
   labels:
-    heritage: {{ .Release.Service }}
-    release: {{ .Release.Name }}
-    app.kubernetes.io/managed-by: {{.Release.Service | quote }}
-    app.kubernetes.io/instance: {{.Release.Name | quote }}
-    app.kubernetes.io/name: {{ template "name" . }}
-    helm.sh/chart: "{{.Chart.Name}}-{{.Chart.Version}}"
     name: portworx
```

Comment:
can we use {{.Release.Name | quote }} here instead of hard coding portworx?

Reply:
We want to be able to use a label selector for the portworx pods. If we use .Release.Name the label can have any random name. With this, we can do `kubectl get pods -n kube-system -l name=portworx` to get the pods.

And `app.kubernetes.io/instance` already has that value set if we want to use it.